app/interactors/book_suggestions_by_genre.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class BookSuggestionsByGenre
+  include Interactor
+
+  def call
+    genre = Book.find(context.book_id).genre
```

Comment:
Try handling errors that might happen here, i.e no book is found with the id that was passed